tests/comments.test.js

```diff
@@ -0,0 +1,186 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const mongoose = require('mongoose');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const mongoose = require("mongoose");
```